src/class/ServerManager/ServerManager.cpp

```diff
@@ -0,0 +1,157 @@
+
+#include <ServerManager/ServerManager.hpp>
```

Comment:
func should be in camelCase

these func are not part of a namespace or a class